src/components/ViewAdmin.vue

```diff
@@ -47,6 +47,12 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 					@update:checked="onChange">
 					{{ 'GPT4All Falcon' }}
 				</NcCheckboxRadioSwitch>
+				<NcCheckboxRadioSwitch :checked.sync="settings['model']"
+					type="radio"
+					value="leo-hessianai"
+					@update:checked="onChange">
+					{{ 'Leo Hessianai' }}
```

Comment:
Maybe we can mention here that it's bilingual German/English. Also we should add it to the README and info.xml